cc.craftos_pc.craftos_pc.yaml

```
@@ -0,0 +1,118 @@
+app-id: cc.craftos_pc.craftos_pc
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: craftos
+
+cleanup:
+  - '/include'
+  - '*.cmake'
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: poco
+    buildsystem: cmake
```

```suggestion
    buildsystem: cmake-ninja
```

(unless it doesn't work)